app/next-client-app/components/core/menubar.tsx

```diff
@@ -1,17 +1,16 @@
 import Link from "next/link";
 import Image from "next/image";
-
 import { SidebarButton } from "./sidebar-button";
 import { sidebarItems } from "./menuItems";
 import { Sidebar } from "./sidebar";
 import { ModeToggle } from "./mode-toggle";
+import { LogOut } from "lucide-react";
 
-export const MenuBar = ({ userName }: { userName: string }) => {
+export const MenuBar = ({ userIsLogged }: { userIsLogged: boolean }) => {
```

Comment:
```suggestion
export const MenuBar = ({ userLoggedIn }: { userLoggedIn: boolean }) => {
```
userIsLogged is verbose but not precise enough. you also used `userLoggedIn` in the sidebar